docs/documentation/pages/internal/DEVELOPMENT.md

```diff
@@ -276,6 +279,21 @@ The helper gets the global context and the desired strategy as the input to set
     value: "system"
   ```
 
+* For the `monitoring-longterm` strategy, the rules will look as follows:
+
+  ```yaml
+  tolerations:
+  - key: dedicated.deckhouse.io
+    operator: Equal
+    value: {{ .Chart.Name }}-longterm
```

Comment:
It is weird to see `{{ .Chart.Name }}` here, ptobably, there was an error before.

Reply:
There was the same as for monitoring strategy as 
  ```yaml
  tolerations:
  - key: dedicated.deckhouse.io
    operator: Equal
    value: {{ .Chart.Name }}
  - key: dedicated.deckhouse.io
    operator: Equal
    value: {{ strategy_name }}
  - key: dedicated.deckhouse.io
    operator: Equal
    value: "system"
  ```

is not it?

Reply:
Section removed with approach. Please review